### ISSUE TITLE
Blacklist keeps resetting

### ISSUE THREAD

**Issue by Jessland86** [LegionDark/Issues#1480]
**Date & Time**: 
1252 EDT

**FFXI Client Version (use `/ver`)**: 
30170304_1

**Server's Expected Client Version matches, yes/no? (use `$ecv`)**: 
yes

**Character Name**: 
Immortality/Lezeattacos, potentially all the rest

**Nation**: 
Sandy

**Job(level)/Sub Job(level)**: 
Varies

**NPC or Monster or item Name**: 
N/A

**Zone name**: 
Varies

**Coordinates (use `$where`)**: 
N/a

**ffxiah.com link (for items issues only)**: 

**Multi-boxing? (multiple clients on same connection)**: 
Yes

**Steps To Reproduce / any other info**: 
Blacklist a ton of people, say an entire linkshells' worth, names seem to be saved correctly on the server side because I cannot add them back to my blacklist since they are already on it, but they do not show up in my list, and even though they are saved correctly they can still send messages/invites/tells.

**Comment by R3P0FFXI** [comment]
Dsp does not store them so the server never saves. May need @atomos to
build  a ashita plug-in for this

**Comment by TeoTwawki** [comment]
wrong, server is *supposed* to store them to a db table. Bringing it up with DSP myself.

**Comment by TeoTwawki** [comment]
`https://github.com/DarkstarProject/darkstar/issues/3874`

**Comment by atom0s** [comment]
The server does store them, however, it is hitting the packet size cap and not handling names beyond the cap. When I coded blacklist for DSP I had no way to test against 100+ names to see how retail handled multiple packets for the blacklist since it is sent as part of the zone-in packet to stop downloading. See here:
https://github.com/DarkstarProject/darkstar/blob/master/src/map/packets/stop_downloading.cpp

Someone needs to go on retail, blacklist a ton of people to go past the packet size cap and capture the packets sent upon zoning to see how retail handles large blacklists.
